db/migrate/20240318105241_add_content_id_to_local_authorities.rb

```diff
@@ -0,0 +1,14 @@
+class AddContentIdToLocalAuthorities < ActiveRecord::Migration[7.1]
+  def up
+    add_column :local_authorities, :content_id, "char(36)"
```

Comment:
Given this is Postgres and I'm assuming we're unlikely to ever migrate to anything else, using the native UUID type might make more sense. We could even have Postgres generate it for us if we wanted to:

```ruby
enable_extension "pgcrypto"

add_column :local_authorities, :content_id, :uuid, default: "gen_random_uuid()"
```

(code off the top of my head so do double check it works!)

That way we don't need to think about generating/assigning it app-side.

Reply:
Yeah, I wasn't sure if we wanted to get yet another type of UUID into publishing API, but those PG ones are just UUID v4, which is what SecureRandom is, and it allows us to get rid of some model-level validation since it'll be enforced by the database anyway.

Done